pkg/daemon/daemon.go

```diff
@@ -734,8 +734,9 @@ func (dn *Daemon) restartDevicePluginPod() error {
 
 	var podToDelete string
 	pods, err := dn.kubeClient.CoreV1().Pods(namespace).List(context.Background(), metav1.ListOptions{
-		LabelSelector: "app=sriov-device-plugin",
-		FieldSelector: "spec.nodeName=" + vars.NodeName,
+		LabelSelector:   "app=sriov-device-plugin",
+		FieldSelector:   "spec.nodeName=" + vars.NodeName,
+		ResourceVersion: "0",
```

Comment:
as we just care for the existance of such pod i believe we are OK with specifying this.

some additional reading on the semantics of ResourceVersion field for other reviewers[1]

[1] https://kubernetes.io/docs/reference/using-api/api-concepts/#semantics-for-get-and-list

Reply:
@adrianchiris thanks for linking the documentation.